pages/faq.tsx

```diff
@@ -128,6 +128,10 @@ const questions: ReadonlyArray<{
         <li>
           Then, run <code>ironfish start</code> and leave that process running.
         </li>
+        <li>
+          Finally, be sure to update your node within one week after the release
```

Comment:
Maybe add command on how to update? Also we might want to tell them how to know when there are updates